.stylelintrc.json

```diff
@@ -18,5 +18,5 @@
     },
```

Comment:
### Your Linters
![first hello microverse linters](https://user-images.githubusercontent.com/93039359/177015917-ecfac7bb-cd34-4c0f-9e67-79fc7a5e1790.png)

- [ ] Your linters are not working. You didn't add linters correctly. Please add them properly according to the [instructions](https://github.com/microverseinc/linters-config/tree/master/html-css). :+1:

Reply:
@HAMMAS-SALEEM  There seems to be an error in Linters / Stylelint (pull_request)  I am not able to see any kind of errors in that